test_simple.py

```diff
@@ -46,6 +47,10 @@ def parse_args():
     parser.add_argument("--no_cuda",
                         help='if set, disables CUDA',
                         action='store_true')
+    parser.add_argument("--pred_depth",
+                        help='if set, predicts metric depth instead of disparity. (This only '
+                             'makes sense for stereo-trained models).',
```

Comment:
Should we also specify that this is only for KITTI?

Reply:
Good idea, done